### PR TITLE
enumeration for osw

### PR DESCRIPTION
**Pull request by iAKM**

### CODE REVIEW

**Review by MashB — DISMISSED** [review]

File: src/environment/environment.ts [hunk 1]
```diff
@@ -22,5 +22,8 @@ export const environment = {
         port: parseInt(process.env.POSTGRES_PORT ?? "5432"),
     },
     appPort: parseInt(process.env.PORT ?? "8080"),
-    authPermissionUrl: process.env.AUTH_PERMISSION_URL
+    authPermissionUrl: process.env.AUTH_HOST+'/api/v1/hasPermission',
+    serviceUrl: process.env.SERVICE_URL,
```
Comment (MashB) [inline src/environment/environment.ts:26]: serviceUrl not required

Comment (MashB) [reply src/environment/environment.ts:26]: Missing package.json command to run unit tests

**Review by MashB — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -105,4 +105,20 @@ sequenceDiagram
     osw-database->>+osw-dataservice:Result
     osw-dataservice->>+Gateway:List of OSW
     Gateway->>+Client: OSW files list
+```
+
+
+## How to run integration test
+To run integration test you need a .env file which will be available on request.
+
```
Comment (MashB) [inline README.md:113]: Please add table with Env variable dependency to run the integration test

**Review by MashB — CHANGES_REQUESTED** [review]

File: src/environment/environment.ts (hunk 1 above)
Comment (MashB) [reply src/environment/environment.ts]: enable disable mock changes what we did for flex is missing

Comment (iAKM) [reply src/environment/environment.ts]: removed serviceUrl

updated commands in package.json to run unit and integration tests

Comment (iAKM) [reply src/environment/environment.ts]: added test_env condition for enable disable mock changes

File: README.md (hunk 2 above)
Comment (iAKM) [reply README.md:113]: added

**Comment by iAKM** [comment]
raised a PR by branching out from this and added EventBus refactoring to that PR

**Pull request closed without merging**